domenetjenester/brevbestiller/src/main/java/no/nav/foreldrepenger/melding/brevbestiller/impl/DokumentFellesDataMapper.java

```diff
@@ -53,11 +53,6 @@ void opprettDokumentDataForBehandling(Behandling behandling, DokumentData dokume
 
         final AktørId søkersAktørId = domeneobjektProvider.hentFagsakBackend(behandling).getAktørId();
 
-        if (!harLenkeForVerge(behandling)) {
```

Comment:
jeg kjenner ikke historikken her - men vil ikke disse uavklarte vergene lenger oppstå? Merkelig kode, men antar det var en grunn for hvordan det ble gjort slik...

Reply:
Det var en unødvendig sjekk - man får akkurat samme effekten fra testen i if (vergeOpt.isEmpty()) {
Dette fordi hvis verge-lenken ikke finnes så returnerer denne Optional.empty()
    public Optional<VergeDto> hentVergeHvisfinnes(List<BehandlingResourceLink> resourceLinker) {
        return resourceLinker.stream()
                .filter(dto -> "verge-backend".equals(dto.getRel()))
                .findFirst().flatMap(link -> hentDtoFraLink(link, VergeDto.class));
    }

FPsak har kode som bare legger til verge-lenke hvis det finnes aksjonspunkt
    if (behandlingHarVergeAksjonspunkt(behandling)) {
            dto.leggTil(get(PersonRestTjeneste.VERGE_PATH, "soeker-verge", uuidDto));   
            dto.leggTil(get(PersonRestTjeneste.VERGE_BACKEND_PATH, "verge-backend", uuidDto)); <------ Formidling 
        }

Reply:
ja, jeg er enig at det var unødvendig kode, bare stusset på at det var gjort slik i utgangspunktet. Men antagelig er det bare ikke sett at det var unødvendig av de som har jobbet med det tidligere.